nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss

```diff
@@ -393,7 +393,7 @@ $warn-dark-palette: (
 );
 
 // Define the palettes for your theme
-$material-primary-light: mat.define-palette($material-primary-light-palette);
+$material-primary-light: mat.define-palette($material-primary-light-palette, 600, 100, 900);
```

Comment:
What is the purpose of this change?

Reply:
The fix is meant to alleviate some of the ambiguity between the disabled action and the cancel actions. This change pushes the primary color here 1 shade darker to increase contrast a bit. You can see the contrast difference in the screens below.

## Before
![image](https://github.com/apache/nifi/assets/638529/83036dc3-ebf2-44c4-b144-8bfb16e38f09)![image](https://github.com/apache/nifi/assets/638529/31fc07e7-bd21-4b74-b486-cdb0141105c6)

## After
![image](https://github.com/apache/nifi/assets/638529/70bd52f6-3039-4f55-9d61-19444e8f9059)![image](https://github.com/apache/nifi/assets/638529/aa0763fd-44c0-4ca5-8159-ad0de5cadeac)